docs/src/pages/docs/getting-started/index.astro

```diff
@@ -9,54 +9,24 @@ const title = 'Getting Started';
-    > which is an alternative design token system, but differs as it predates the new standard (that Cobalt uses). The new design tokens standard is the <a
-      href="https://github.com/design-tokens/community-group#companies-and-open-source-projects-represented-on-the-dtcg"
-      target="_blank"
-      rel="noopener noreferrer">largest collaborative format</a
-    > to-date, involving input from Figma, Adobe, Salesforce, Google, and more to push the format forward.
-  </p>
-
-  <p>Figma has a great video explaining what an exciting leap forward this is:</p>
+  <Code lang="shell" code={`$ npx co build
+  ✔  180 tokens built 100ms`} />
 
-  <div class="yt-embed">
-    <iframe
-      width="560"
-      height="315"
-      src="https://www.youtube-nocookie.com/embed/ssOdzxZdg58"
-      title="YouTube video player"
-      frameborder="0"
-      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
-      allowfullscreen></iframe>
-  </div>
+  <p>Cobalt turns your <a href="/docs/guides/design-tokens">design tokens</a> into code using a CLI or Node.js. Cobalt is configurable and pluggable, and can generate <a href="/plugins/js">JavaScript</a>, <a href="/plugins/js">TypeScript</a>, <a href="/plugins/js">JSON</a>, <a href="/plugins/css">CSS</a>, and <a href="/plugins/js">Sass</a> via the official <a href="/docs/plugins">plugins</a>. Or you can generate anything else by <a href="/docs/guides/plugins">creating your own plugin</a>.</p>
 
-  <h2>Workflow</h2>
-
-  <p>Starting with your design tokens (which will usually be in a program like Figma, or whatever you use to manage your design system), working with Cobalt involves the following steps:</p>
+  <p>Using Cobalt is as easy as:</p>
 
   <ol>
-    <li>Creating a <code>tokens.json</code> design tokens file</li>
-    <li>Installing Cobalt output plugins of your choice (e.g. JS, TS, CSS, and/or Sass)</li>
-    <li>Running the CLI to output all your desired formats</li>
-    <li>(Bonus) Adding Cobalt to CI to always keep your design system up-to-date</li>
+    <li>Creating a <code>tokens.json</code> file</li>
+    <li>Installing Cobalt plugins</li>
+    <li>Running the CLI
+    <li><i>(Optional)</i> Adding Cobalt to CI</li>
   </ol>
 
-  <h3>1. Creating the `tokens.json` file</h3>
+  <h2>1. Creating the <code>tokens.json</code> file</h2>
 
   <p>
-    This is the boring part—converting your tokens to JSON. Usually this involves manually pulling values from your styleguide or your design program (like Figma) and formatting the JSON yourself. There’s not a great UI for this yet (feel
+    Converting your tokens to JSON is a manual process of grabbing values from your design files and formatting the JSON yourself. There’s not a great UI for this yet (feel
     free to <a href="https://github.com/drwpow/cobalt-ui/issues">suggest one</a> if you have one you like), but <strong>the Cobalt team is developing one currently</strong>.
```

Comment:
> the Cobalt team is developing one currently

This line has been in the docs for a while. But it’s still true! 🙂 This has been on the backburner for a bit, but has been my highest focus for the past few months. I’m excited to announce it when it’s ready.